backend/api/v2beta1/error.proto

```diff
@@ -15,17 +15,27 @@
 syntax = "proto3";
 
 option go_package = "github.com/kubeflow/pipelines/backend/api/v2beta1/go_client";
-package v2beta1;
+package kubeflow.pipelines.backend;
 
 import "google/protobuf/any.proto";
 
+// Error message.
 message Error {
+  // Error message.
   string error_message = 1;
+
+  // Error details.
   string error_details = 2;
 }
 
+// Status message representing result of a request. 
 message Status {
+  // Error message.
   string error = 1;
```

Comment:
Not your change, but was this meant to be `Error error = 1;`? If its only the error message, then maybe name it `string error_message = 1;` (to be consistent with line 25).